packaging/rpm-ostree.spec

```diff
@@ -218,6 +224,9 @@ sed -i -e '/https:\/\//d' cargo-vendor.txt
 %if %{without ostree_ext}
 rm -vrf $RPM_BUILD_ROOT/usr/libexec/libostree/ext
 %endif
+%if %{without kernel_install}
```

Comment:
Somehow this is ending up running on F42... 
```
bash-5.2# ls /usr/lib/kernel/install.d/
20-grub.install    50-dracut.install  90-loaderentry.install  92-crashkernel.install
50-depmod.install  60-kdump.install   90-uki-copy.install     99-grub-mkconfig.install
bash-5.2# rpm -qa rpm-ostree
rpm-ostree-2025.3-1.fc42.x86_64
bash-5.2# 

```

Reply:
bcond_without I think is what we needed I missed this.

Reply:
I removed the the conditional for now on rawhide, need to investigate with the copr builds before syncing the specs again.

Reply:
and in f41 it adds it.. the logic is reverse somehow... sad part is I don't see it.

```
bash-5.2# cd /usr/lib/kernel/install.d/
bash-5.2# ls
05-rpmostree.install  50-depmod.install  60-kdump.install	 90-uki-copy.install	 99-grub-mkconfig.install
20-grub.install       50-dracut.install  90-loaderentry.install  92-crashkernel.install
bash-5.2# rpm -qa rpm-ostree
rpm-ostree-2025.3-1.fc41.x86_64
```

Reply:
https://rpm-software-management.github.io/rpm/manual/conditionalbuilds.html
```
Using %bcond_with and %bcond_without
Build conditionals can also be created using the macros %bcond_with and %bcond_without. These macros create command line options. When you add an option to build with feature X, it implies that the default is to build without.
```
So confusing